components/scream/src/share/field/field_utils.hpp

```diff
@@ -1,7 +1,9 @@
 #ifndef SCREAM_FIELD_UTILS_HPP
 #define SCREAM_FIELD_UTILS_HPP
 
-#include "field.hpp"
+#include "share/field/field.hpp"
```

Comment:
On second thought, maybe `unflatten` could go here as a free function, but we'd have to add the field or layout as an arg.

Reply:
Could also be a (kokkos-friendly) method of `FieldLayout`.

Reply:
I went for this file, with some help from FieldLayout.